packages/page-staking/src/react-components/FinalizerPerformance/index.tsx

```diff
@@ -12,13 +12,23 @@ interface Props {
   filterName: string;
   scoresEnabled: boolean;
   abftScore?: number,
+  session?: number,
 }
 
 function queryAddress (address: string) {
   window.location.hash = `/staking/query/${address}`;
 }
 
-function FinalizerAddress ({ abftScore, address, filterName, scoresEnabled }: Props): React.ReactElement<Props> | null {
+/**
+ * A reusable component describing ABFT performance of a finalization committee member in a session.
+ * @param abftScore ABFT performance, a distance between this member top round and chain-wide top round
+ * @param address Finalizer account id
+ * @param filterName Filter string from parent component, can be empty
+ * @param scoresEnabled A boolean flag denoting whether ABFT scores are supported on-chain
+ * @param session session number (optional). If given, additional column will be rendered
+ * @constructor
```

Comment:
I'm not sure what's up with `@constructor`

Reply:
My IDE proposed it but indeed it's not needed, removed.